### ISSUE TITLE
Add shortcuts for "previous (unreviewed) diff"

### ISSUE THREAD

**Issue by sfc-gh-lmoll** [MarcinZukowski/review-pal#4]
There are keyboard shortcuts for the next diff:
shift+K - next unreviewed diff
shift+ctrl+K - next diff

But I'm missing their counterparts to navigate backwards.

**Comment by MarcinZukowski** [comment]
Yeah, all these next/prev are not necessarily easy, but I'll try when I find time :)
